<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormLancarCredito</class>
 <widget class="QMainWindow" name="FormLancarCredito">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastrar Produto</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../img/listar produtos.png</normaloff>../../img/listar produtos.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Lançar Crédito</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCredito">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Crédito:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="creditoSpinBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="prefix">
     <string/>
    </property>
    <property name="suffix">
     <string/>
    </property>
    <property name="maximum">
     <double>999.990000000000009</double>
    </property>
   </widget>
   <widget class="QLabel" name="labelCodVenda">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cód. Venda:</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnLancar">
       <property name="text">
        <string>Lançar</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../img/credit.png</normaloff>../../img/credit.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLimpar">
       <property name="text">
        <string>Limpar</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../img/borracha.png</normaloff>../../img/borracha.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="editCodVenda">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="editCliente">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelCliente">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cliente:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editValorVenda">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>100</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelValorVenda">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Valor da Venda:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelValorVenda_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Valor Pago:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editValorVenda_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Normal</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>editCodVenda</tabstop>
  <tabstop>creditoSpinBox</tabstop>
  <tabstop>btnLancar</tabstop>
  <tabstop>btnLimpar</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnLimpar</sender>
   <signal>clicked()</signal>
   <receiver>editCodVenda</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLimpar</sender>
   <signal>clicked()</signal>
   <receiver>creditoSpinBox</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
